apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js

```diff
@@ -69,9 +68,20 @@ function BrowseIdeas() {
     setCards(cardsClone);
   };
 
-  React.useEffect(async () => {
-    const ideaCards = cleanDataList(await agent.Ideas.get(
-      new URLSearchParams(`populate=*&pagination[pageSize]=1000`)));
+  React.useEffect(() => {
+    axios
+      .get(`${process.env.NEXT_PUBLIC_STRAPI_URL}/idea-cards`, {
```

Comment:
can this be reverted? We are making it a requirement within ideaspace to use the agent for all request to the backend.